core/src/elements/ons-pull-hook.js

```diff
@@ -173,9 +175,11 @@ export default class PullHookElement extends BaseElement {
     // Hack to make it work on Android 4.4 WebView and iOS UIWebView. Scrolls manually
```

Comment:
We are only applying this for iOS now, it seems, so it's worth updating this comment to reflect that

Reply:
This one was also necessary for Android, it's a different issue from the iOS page scroll flickering. Actually I could not reproduce it in my Android 7 but I remember seeing the issue in older versions some time ago, so it's safer to keep it.

Reply:
I've just realised I mis-read the line `this._shouldFixScroll = util.globals.actualMobileOS !== 'other';` as `actualMobileIOS`. Ignore this comment, and also the next one!